server/graphql/v2/mutation/TierMutations.js

```diff
@@ -73,11 +73,13 @@ const tierMutations = {
         tierUpdateData.data.invoiceTemplate = args.tier.invoiceTemplate;
       }
 
+      const updatedTier = await tier.update(tierUpdateData);
+
       // Purge cache
       purgeCacheForCollective(collective.slug);
       purgeCacheForPage(`/${collective.slug}/contribute/${tier.slug}-${tier.id}`);
 
-      return await tier.update(tierUpdateData);
+      return updatedTier;
```

Comment:
The reason I did this change was that I noticed that otherwise when the tier creation or tier edit modal is saved you need to do a page refresh sometimes to notice the changes. This is because previously we were doing the update after the cache is cleared.